Release Note for WebView2 with Stable 105 and Pre-release 106 (#2153)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/release-notes.md

<!-- ====================================================================== -->

> ## 1.0.1369-prerelease

Reviewer: @imicy Will there be a release H2 section in this PR?

Reviewer 2: https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1343.22 - "a day ago"

> ## 1.0.1369-prerelease

Reviewer: @nishitha-burman Is the overview for the api articles overview need to be updated.  Will it be in a seperate pr?    Will we need to update it?

Reviewer 3: Yes, that will be in a separate PR. @imicy

Reviewer: Sweet.  Please add me to the pr.

> Release Date: Sep 4, 2022

Reviewer 4: ```suggestion
Release Date: September 4, 2022

```

[NuGet package for WebView2 SDK 1.0.1369-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1369-prerelease)

For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 106.0.1369.0 or higher. 

### General

#### Promotions

The following items are now stable:

*  The drag and drop API: 
   * `DragEnter`
   * `DragLeave`
   * `DragOver`
   * `Drop`

##### [.NET/C#](#tab/dotnetcsharp)

> * [CoreWebView2CompositionController Class](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller?view=webview2-dotnet-1.0.1369-prerelease&preserve-view=true)

Reviewer 4: ```suggestion
* [CoreWebView2CompositionController.DragLeave Method](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave?view=webview2-dotnet-1.0.1369-prerelease#microsoft-web-webview2-core-corewebview2compositioncontroller-dragleave)

```

##### [WinRT/C#](#tab/winrtcsharp)

> * [CoreWebView2CompositionController Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.1369-prerelease&preserve-view=true)

Reviewer 4: ```suggestion
* [CoreWebView2CompositionController.DragLeave Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.1369-prerelease&preserve-view=true#dragleave)

```
Will there be now the other 3 methods, for .NET & WinRT?  In addition to DragLeave, Win32 also has DragEnter, DragOver, and Drop:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true

##### [Win32/C++](#tab/win32cpp)

> * [ICoreWebView2CompositionController3 Class](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true)

Reviewer 4: ```suggestion
* [ICoreWebView2CompositionController3.DragEnter method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragenter)
* [ICoreWebView2CompositionController3.DragLeave method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragleave)
* [ICoreWebView2CompositionController3.DragOver method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragover)
* [ICoreWebView2CompositionController3.Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#drop)

```

---

#### Bug fixes

> *   Fix a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))

Reviewer 4: ```suggestion
*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))

```

*   Fixed a bug that produced simultaneous WebView creation failure. This change is Runtime-specific. ([Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703))  

*   Fixed print settings paper size to support dimensions as small as 0.01 inches (Runtime).  

> *   Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))  

Reviewer 4: ```suggestion
*   Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))  
```

> 

Reviewer: Rendered page for review:

https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2153

<!-- ====================================================================== -->
## 1.0.1343.22

Release Date: September 6, 2022

[NuGet package for WebView2 SDK 1.0.1343.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1343.22)

For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 105.0.1343.22 or higher.

> ### General

Reviewer 2: ```suggestion
### General

No changes in this release.

```

> * Fixed a bug in the wv2winrt tool where a WinMD file wasn't referenced in some projects.

Reviewer 2: ```suggestion
* Fixed a bug in the **wv2winrt** tool where a WinMD file wasn't referenced in some projects.

```

## 1.0.1369-prerelease

Release Date: September 6, 2022

[NuGet package for WebView2 SDK 1.0.1369-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1369-prerelease)

For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 106.0.1369.0 or higher. 

### General

> * Fixed a bug that produced simultaneous WebView creation failure (Runtime) [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)

Reviewer 2: ```suggestion
* Fixed a bug that produced simultaneous WebView creation failure (Runtime). [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)

```

* [ICoreWebView2CompositionController3.DragLeave method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragleave)
* [ICoreWebView2CompositionController3.DragOver method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragover)
* [ICoreWebView2CompositionController3.Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#drop)

---

#### Bug fixes

*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))

*   Fixed a bug that produced simultaneous WebView creation failure. This change is Runtime-specific. ([Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703))  

*   Fixed print settings paper size to support dimensions as small as 0.01 inches (Runtime).  

> *   Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))  

Reviewer 2: My rewrite got lost.  Checking past comments...

> *   Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))  

Reviewer 2: ```suggestion
*   Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))

```

Reviewer 2: @JasonAndrewWriter  - this change was lost, i will re-enter 3rd time.  trying to change from:

* Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))

to:

* Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))

##### [Win32/C++](#tab/win32cpp)

* [ICoreWebView2CompositionController3.DragEnter method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragenter)
* [ICoreWebView2CompositionController3.DragLeave method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragleave)
* [ICoreWebView2CompositionController3.DragOver method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragover)
* [ICoreWebView2CompositionController3.Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#drop)

---

#### Bug fixes

*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))

> *   Fixed a bug that produced simultaneous WebView creation failure (Runtime) [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)

Reviewer 2: ```suggestion
*   Fixed a bug that produced simultaneous WebView creation failure (Runtime). [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)

```

* [ICoreWebView2CompositionController3.DragLeave method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragleave)
* [ICoreWebView2CompositionController3.DragOver method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragover)
* [ICoreWebView2CompositionController3.Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#drop)

---

#### Bug fixes

*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))

*   Fixed a bug that produced simultaneous WebView creation failure (Runtime). [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)

*   Fixed print settings paper size to support dimensions as small as 0.01 inches (Runtime).  

> *   Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))

Reviewer 2: ```suggestion
* Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))
```
3rd entry of this Suggestion
